src/main/scala/org/constellation/lb/Manager.scala

```diff
@@ -38,48 +42,74 @@ class Manager(init: NonEmptyList[Addr], config: LoadbalancerConfig)(
   private lazy val hostsRef =
     Ref.unsafe[IO, NonEmptyMap[Addr, Option[List[Info]]]](init.map(addr => addr -> None).toNem)
 
+    private lazy val sessionRef =  Ref.unsafe[IO, Option[Long]](None)
```

Comment:
Looks like you have reformatted code that you haven't touched but here for example is formatted wildly (2 spaces after `=`)? If you want to reformat the code using scalafmt or anything, would be great to make it as a totally separate commit. Otherwise, it's hard to figure out what is a real change here

Reply:
fixed with scalafmt